docs/best-practices/testing/how-to-use-hypernode-docker.md

```diff
@@ -0,0 +1,145 @@
+---
+myst:
+  html_meta:
+    description: 'Docker image can be used to set up a fast and easy local development
+      environment for Hypernode. See intstructions that guide you through the process. '
+    title: How to use hypernode docker?
+redirect_from:
+  - /en/hypernode/hypernode-docker/how-to-use-hypernode-docker/
+---
+
+# Usage of hypernode-docker
```

Comment:
I think `Usage of Hypernode Docker` makes more sense

Reply:
Remove the doc, it's the same as current Hypernode Docker doc